climada/engine/forecast.py

```diff
@@ -526,15 +531,20 @@ def _plot_imp_map(
         return fig, axis_sub
 
     def plot_hist(
-        self, run_datetime=None, save_fig=True, close_fig=False, figsize=(9, 8)
+            self, run_datetime=None, explain_str=None, save_fig=True,
+            close_fig=False, figsize=(9, 8)
     ):
```

Comment:
Indentation is off here. Number of arguments also warrants line breaks.
```suggestion
        self,
        run_datetime=None,
        explain_str=None,
        save_fig=True,
        close_fig=False,
        figsize=(9, 8),
    ):
```